lib/reporting/authentication_report.rb

```diff
@@ -30,11 +30,14 @@ def self.all_events
 
     # @param [String] isssuer
     # @param [Range<Time>] time_range
-    def initialize(issuer:, time_range:, verbose: false, progress: false)
+    def initialize(issuer:, time_range:, verbose: false, progress: false, slice: 3.hours,
```

Comment:
@zachmargolis while i was doing this PR i noticed that the default for progress here is `false` but in the command line options, the default is `true`. was that on purpose? i can update this default if it was an accident, but wanted to confirm before changing it!

Reply:
I forget, but I think my process was: if using the command line options, this was definitely a thing we'd want progress updates for, but if toying around in the console or even maybe if we ever used the code directly in a background job, the logging would be annoying, so that's why it defaulted to off there

Reply:
that makes sense! i was wondering if that was the thinking behind it. thanks for the clarification!